docs/plugin-authoring-evaluator.md

```diff
@@ -61,23 +61,22 @@ function getDeliciousnessPrompt(ai: Genkit) {
       output: {
         schema: DeliciousnessDetectionResponseSchema,
       }
-    },
-    `You are a food critic. Assess whether the provided output sounds delicious, giving only "yes" (delicious), "no" (not delicious), or "maybe" (undecided) as the verdict.
+      prompt: `You are a food critic. Assess whether the provided output sounds delicious, giving only "yes" (delicious), "no" (not delicious), or "maybe" (undecided) as the verdict.
```

Comment:
Reduce indentation?

Reply:
The indentation matches the new prompt API, it's nested one level deeper in the config object.